### ISSUE TITLE
[MU4 Issue] Accessibility: no NVDA feedback within score options panel

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#12807]
**Describe the bug**
NVDA users cannot select the key or time signature for their scores, or the tempo or length, using the wizard, because screen reader does not reader the contents of these options.  Narrator works, however.

**To Reproduce**
1. Start NVDA
2. Create new score for any instruments
3. In the "Additional score information" panel, select the key signature control
4. Use keyboard to navigate through the keys

**Result**
Nothing is read aloud

**Expected behavior**
Each key should be read as you navigate

**Platform information**
 - OS: Windows / NVDA

**Additional context**
Same for the time signature, tempo, and measures control

**Comment by DmitryArefiev** [comment]
Fixed in #13013

**Comment by DmitryArefiev** [comment]
But found the same occurs on Linux  (logged #13311)
